applications/crossbar/src/modules/cb_search.erl

```diff
@@ -411,7 +410,7 @@ get_end_key(Context, Type, Value) ->
 %%--------------------------------------------------------------------
 -spec next_binary_key(binary()) -> ne_binary().
 next_binary_key(<<>>) ->
-    <<"\ufff0">>;
+    <<0>>;
```

Comment:
Are you sure about this one? It's supposed to be the biggest possible string. Isn't `<<0>>` considered the smallest?
Actually is this valid?